mix.lock

```diff
@@ -55,7 +55,7 @@
   "parse_trans": {:hex, :parse_trans, "3.3.0", "09765507a3c7590a784615cfd421d101aec25098d50b89d7aa1d66646bc571c1", [:rebar3], [], "hexpm"},
   "phoenix": {:hex, :phoenix, "1.4.7", "e5bc9adcfa4c75a60d0952817d015a6caecc2a8812f4c53c1b61776dfa91d8ce", [:mix], [{:jason, "~> 1.0", [hex: :jason, repo: "hexpm", optional: true]}, {:phoenix_pubsub, "~> 1.1", [hex: :phoenix_pubsub, repo: "hexpm", optional: false]}, {:plug, "~> 1.8.1 or ~> 1.9", [hex: :plug, repo: "hexpm", optional: false]}, {:plug_cowboy, "~> 1.0 or ~> 2.0", [hex: :plug_cowboy, repo: "hexpm", optional: true]}, {:telemetry, "~> 0.4", [hex: :telemetry, repo: "hexpm", optional: false]}], "hexpm"},
   "phoenix_pubsub": {:hex, :phoenix_pubsub, "1.1.2", "496c303bdf1b2e98a9d26e89af5bba3ab487ba3a3735f74bf1f4064d2a845a3e", [:mix], [], "hexpm"},
-  "plasma_contracts": {:git, "https://github.com/omisego/plasma-contracts", "1608296ef56ab98d857321a687c10dcdd86d72d3", [branch: "master", sparse: "plasma_framework/contracts/"]},
+  "plasma_contracts": {:git, "https://github.com/omisego/plasma-contracts", "e3c15df7c816770c60d257557c62b1f7105e07c2", [branch: "master", sparse: "plasma_framework/contracts/"]},
```

Comment:
Another one is:
- (same repository) https://github.com/omisego/elixir-omg/blob/master/Dockerfile.plasma_contracts#L15
- (omisego-images) https://github.com/omisego-images/docker-elixir-omg/blob/master/tester/Dockerfile.plasma_deployer#L15 *after which you can bump docker-compose reference*:
https://github.com/omisego/elixir-omg/blob/master/docker-compose.yml#L19

Reply:
> (same repository) https://github.com/omisego/elixir-omg/blob/master/Dockerfile.plasma_contracts#L15

Might as well remove the entire file since it's no longer used (I think?)

Reply:
Yes I don't think it's being used.